README.md

```diff
@@ -91,3 +93,57 @@ The strategy active by default is the one specified in the `defaultStrategy` ent
 | --pause           | configure       | temporarily disable the service |
 | --resume          | configure       | resume the service              |
 
+# NixOS
```

Comment:
Yes and no.
Most flakes are standalone Git Repositorys or on [FlakeHub](https://flakehub.com/) (for exmple [my Flake](https://flakehub.com/flake/Svenum/Solaar-Flake)).
Flake Hub would be the next step (my opinion), because you need an FlakeHub Account and another GitHub Action.
The Repo you pointed to could be targeted, but I try to get my Solaar-Flake there since Sep 2023...

In addition you could try to add this to the main nixos/nixpkgs repo, but there we would not be able to change thinks as quickly as with an own flake.

Reply:
I see, thanks for the clarification